### PR TITLE
Feature/33 jmrtd ocr coupling closes #33

### LINKED ISSUE #33: Make jMRTD and OCR coupling
When both issues are closed and the code is merged into develop, a coupling can be made between jMRTD and OCR so passport information does not need to be hard-coded.

### PR DESCRIPTION
**Pull request by wkmeijer**
In order to scan the passport first the MRZ has to be scanned by using OCR, if this fails a manual input can be used. The document details get send to the passport connection activity which opens a connection with the passport.

### CODE REVIEW

**Review by jonathanraes — APPROVED** [review]

File: app/src/main/AndroidManifest.xml [hunk 1]
```diff
@@ -31,7 +31,6 @@
 
             <intent-filter>
                 <action android:name="android.intent.action.MAIN" />
-                <category android:name="android.intent.category.LAUNCHER" />
             </intent-filter>
```
Comment (jonathanraes) [inline app/src/main/AndroidManifest.xml]: I think this whole intent-filter can be deleted

Comment (wkmeijer) [reply app/src/main/AndroidManifest.xml]: agreed

File: app/src/main/java/com/digitalvotingpass/digitalvotingpass/MainActivity.java [hunk 2]
```diff
@@ -58,6 +73,7 @@ protected void onActivityResult(int requestCode, int resultCode, Intent data) {
         switch(requestCode) {
             case (GET_MANUAL_DOC_INFO): {
                 if (resultCode == RESULT_OK) {
+                    documentData = (HashMap<String, String>) data.getSerializableExtra("result");
```
Comment (klikooo) [inline app/src/main/java/com/digitalvotingpass/digitalvotingpass/MainActivity.java]: The cases GET_MANUAL_DOC_INFO and GET_DOC_INFO both have the same code in them.

**Review by klikooo — CHANGES_REQUESTED** [review]
Please fix the above comment

**Review by klikooo — APPROVED** [review]

**Pull request merged**